.github/actions/synthetics-stack-e2e/action.yml

```diff
@@ -0,0 +1,155 @@
+---
+name: synthetics-stack-e2e
+description: |
+  An action that orchestrates the Synthetics Stack E2E tests for a Kibana PR.
+  It first creates a Kibana docker image, then it deploys an ESS cluster with the created docker image,
+  and finally it runs the Synthetics Stack E2E tests against the deployed ESS cluster.
+
+inputs:
+  vault-url:
+    description: 'Vault URL'
+    required: true
+  vault-role-id:
+    description: 'Vault role ID'
+    required: true
+  vault-secret-id:
+    description: 'Vault secret ID'
+    required: true
+  comment-id:
+    description: 'The GitHub Comment ID'
+    default: ${{ github.event.comment.id }}
+  commit-sha:
+    description: 'The commit SHA'
+    default: ${{ github.event.pull_request.head.sha }}
+    required: false
+  pr-number:
+    description: 'The PR number'
+    default: ${{ github.event.issue.number }}
+    required: false
+  user:
+    description: 'The GitHub user that triggered the workflow'
+    default: ${{ github.triggering_actor }}
+    required: true
+  context:
```

Comment:
ditto, maybe we can enforce the same commit status name, so we don't let people to change it? or do you see this might be needed?